Backend/Lyzer/Lyzer.http

```diff
@@ -0,0 +1,6 @@
+@Lyzer_HostAddress = http://localhost:5131
+
+GET {{Lyzer_HostAddress}}/weatherforecast/
+Accept: application/json
+
+###
```

Comment:
@Pranav-Zodes-44 Do you know what this is?

Reply:
Asked ChatGPT, apparently it just allows us to run requests directly in VS, instead of using cURL or Postman. Can probably just keep it there, but doubt it we'll need it